### ISSUE TITLE
D1 R32 ILI9486 Failed to start: src/drv/tft/tft_driver_lovyangfx.cpp

### ISSUE THREAD

**Issue by dfsx1** [HASwitchPlate/openHASP#534]
### Perform all steps below and tick them with [x]
- [x] Check the related part of the [Documentation](https://www.openhasp.com/)
- [x] Update openHASP to the latest version
- [x] Reproduce the issue and describe all steps

### Describe the bug

TFT : Failed to start: src/drv/tft/tft_driver_lovyangfx.cpp line 327
```
[19:20:50]        open____ _____ _____ _____
[19:20:50]          |  |  |  _  |   __|  _  |
[19:20:50]          |     |     |__   |   __|
[19:20:50]          |__|__|__|__|_____|__|
[19:20:50]        Home Automation Switch Plate
[19:20:50]        Open Hardware edition v0.7.0-rc8
[19:20:50]
[19:20:50][       0.091][110580/197716 44][    0/    0  0] DBUG: Started @ 115200 bps
[19:20:50]#[       0.100][110580/197716 44][    0/    0  0] DBUG: Environment: d1-r32-unoshield_ili9486_adc
[19:20:50]#[       0.110][110580/197716 44][    0/    0  0] UART: Started
[19:20:50]#[       0.118][110580/197716 44][    0/    0  0] UART: Client login from serial
[19:20:50]#[       0.127][110580/197716 44][    0/    0  0] CONF: Failed to load /config.json
[19:20:50]Prompt > [       0.137][110580/197716 44][    0/    0  0] CONF: Loading EEPROM
[19:20:50]Prompt > [       0.147][110580/197716 44][    0/    0  0] CONF: Loaded EEPROM
[19:20:50]Prompt > [       0.155][110580/197780 44][    0/    0  0] DBUG: Loading debug settings
[19:20:50]Prompt > [       0.166][110580/197780 44][    0/    0  0] GPIO: Loading GUI settings
[19:20:50]Prompt > [       0.176][110580/197780 44][    0/    0  0] HASP: Loading HASP settings
[19:20:50]Prompt > [       0.186][110580/197780 44][    0/    0  0] WIFI: Loading WiFi settings
[19:20:50]Prompt > [       0.204][110580/197084 43][    0/    0  0] MQTT: Loading MQTT settings
[19:20:50]Prompt > [       0.215][110580/197040 43][    0/    0  0] TELN: Loading Telnet settings
[19:20:50]Prompt > [       0.225][110580/197040 43][    0/    0  0] MDNS: Loading MDNS settings
[19:20:50]Prompt > [       0.235][110580/197040 43][    0/    0  0] HTTP: Loading HTTP settings
[19:20:50]Prompt > [       0.245][110580/196996 43][    0/    0  0] GPIO: Loading GPIO settings
[19:20:50]Prompt > [       0.255][110580/196996 43][    0/    0  0] CONF: Settings loaded
[19:20:50]Prompt > [       0.437][110580/155080 28][    0/    0  0] MSGR: Starting...
[19:20:50]Prompt > [       0.447][110580/155260 28][    0/    0  0] MSGR: Started
[19:20:50]Prompt > [       0.455][110580/155260 28][    0/    0  0] TFT : Starting...
[19:20:50]Prompt > [       0.464][110580/155260 28][    0/    0  0] TFT : Starting...
[19:20:50]Prompt > [       0.474][110580/154528 28][    0/    0  0] TFT : Failed to start: src/drv/tft/tft_driver_lovyangfx.cpp line 327
```
### To Reproduce
Run binaries from the webinstallers for d1-r32-unoshield_ili9486_adc
I get the same error with v0.6.3 and v0.7.0

### Expected behavior
When I look at the code I only see ILI9488 inside the `switch(tft_driver){}`
https://github.com/HASwitchPlate/openHASP/blob/e2910604a2ce4d979c47fea352b2d586b5b9ca64/src/drv/tft/tft_driver_lovyangfx.cpp#L296-L305
ILI9486 is missing?
Can someone clarify me if I'm the first person actually testing the ILI9486 while pre-compiled binaries are available for years?

I tried just adding lgfx::Panel_ILI9486() (3e5da3c9c5cb589baf63293a9272a0e7634ac9a3):
After compiling at least now openHASP starts 👍 and I can access over WIFI.
But I still get a white screen and need to do more research.

I also see pins are configured wrong:
- TFT_RST=32 should be TFT_RST=36
- TFT_CS=33 should be TFT_CS=34 but TOUCH_anWR=34 is already set to that pin?!

```
-D TOUCH_anDC=35 ; Analog pin to be connected to TFT_DC
-D TOUCH_anWR=34 ; Analog pin to be connected to TFT_WR
```
I don't understand these comments suggesting connecting analog pins to digital pins.
Pin layout is fixed on this UNO shield, I don't use any wires or modification (yet).
If I leave TOUCH_anWR=34 conflicting with TFT_CS=34 I'm afraid I maybe destroy some IO pins.
Even if this hardware costed me only $4 I dont have any spares at the moment.
Maybe I can try to comment-out these TOUCH pins?
I dont care about touch functionality before I get the display to work...

**Comment by fvanroie** [comment]
IIRC the analog touch driver was code contributed by a user. I don't think there are many users with this hardware combination out there. As such, the combination is tested (often) and relies an user feedback. I'll have a look...

It's always best to start with the display config first and see that those pins are correct.
Touch pins can be set to any of the free pins as to not interfere with the display.

**Comment by dfsx1** [comment]
@fvanroie
After some research I learned ESP32 pins 34,35,36,39 are input pins only.
Hardware modification was needed for R1 D32 as documented here: https://github.com/Bodmer/TFT_eSPI#8-bit-parallel-support

![image](https://github.com/HASwitchPlate/openHASP/assets/60702962/ec045e5f-c84e-4404-b04e-6701e6fbe90e)

I see a screenshot in the analog touch driver contributors pull request #213
Don't understand how @wesleygas used pin 34 for TFT_WR as this is already connected to TFT_CS on the shield.
I chose to use pin 39 instead of 34 for TOUCH_anWR, to keep pin 33=>34 compatible with the mod described in TFT_eSPI docs.

![image](https://github.com/HASwitchPlate/openHASP/assets/60702962/d73efa4f-04c2-4827-81c9-b298dd1515ee)

After the hardware mod (and fix #553 for this error msg) I got the display working.
But with LovyanGFX I only see half of some kind of bootscreen.
Pages don't update and no touch.
(TFT : Touch Failed to start).

![image](https://github.com/HASwitchPlate/openHASP/assets/60702962/e4839bf0-18f0-434d-9653-7628f87be6ef)

TFT_eSPI works fine, only no touch.
(#warning >>>>------>> TOUCH_CS pin not defined, TFT_eSPI touch functions will not be available! [-Wcpp])
I'm wondering if I can get the analog touch driver working for TFT_eSPI 🤔

![image](https://github.com/HASwitchPlate/openHASP/assets/60702962/8d3e0a35-c3e7-408c-8808-c1a9dd97aee6)
